build/tools/templates/SharedFx/SharedFx.csproj

```diff
@@ -23,4 +23,33 @@
    <PackageReference Include="Microsoft.AspNetCore.All" Version="$(PackageVersion)" ExcludeAssets="Build" />
    <PackageReference Include="Microsoft.NETCore.App" Version="$(RuntimeFrameworkVersion)"/>
   </ItemGroup>
+
+  <Target Name="GetPackageDefinitions" Returns="@(_PackageDefinitions)">
+    <ItemGroup>
+      <ProjectAssetsJson Include="$(MSBuildThisFileDirectory)**\project.assets.json" />
```

Comment:
Why search? The property `ProjectAssetsFile` should already be set to the right file path.

https://github.com/dotnet/sdk/blob/b5be57155df09c2f76afb6050397d1a85a745f45/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets#L28

Reply:
Didn't know this existed. I'll update https://github.com/aspnet/Universe/blob/e860022e1a7b1ea825c1d185d1084feec24faf47/build/tools/templates/RS.Manifest/RS.Manifest.csproj#L25 as well.